File: Lexer/DivideToken.cpp
#include "token.h"
#include <string>
using namespace std;

class Divide : public Token {
	bool Token::equals()
	{

	}

	int Token::hashCode()
	{
		return 10;
	}

	string Token::toString()
	{
		return "/";
	}
};

File: src/main/java/parser/DivideToken.java
package parser;

public class DivideToken implements Token{
	public boolean equals(final Object other) {
		return other instanceof DivideToken;
	}
	
	public int hashCode() {
		return 1;
	}
	
	public String toString() {
		return "DivideToken";
	}

}
